rust-k256/src/lib.rs

```diff
@@ -106,7 +105,7 @@ impl PlumeSignature<'_> {
         // don't forget to check `c` is `Output<Sha256>` in the #API
         let c = Output::<Sha256>::from_slice(self.c);
         // TODO should we allow `c` input greater than BaseField::MODULUS?
-        let c_scalar = &Scalar::from_uint_reduced(U256::from_be_byte_array(c.to_owned()));
```

Comment:
I wonder how `clone` is better? Impact is diminishing ofc, but just for the sake of discussion.

Reply:
(I have changed to `to_owned` to ensure that `c` is "eaten" at that point and can't be used further.)

Reply:
Yep. Didn't check if that's used further or not.
Will change it!

Reply:
Hey just now checked that `c` is indeed used later so clone makes sense. But the value to be cloned itself is borrowed one. In that sense, to_owned makes more sense.

Reply:
I guess it's so, was commenting from the top of my head! %)